rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java

```diff
@@ -288,7 +288,16 @@ private ToolStrip buildLogoStrip() {
         logoStrip.addFill();
         logoStrip.addMember(new Img(RMImagesUnbundled.COMPANY_ICON, logoStripHeight, logoStripHeight));
         logoStrip.addFill();
-        logoStrip.addMember(new Img(RMImagesUnbundled.AE_LOGO, 146, logoStripHeight));
+
```

Comment:
Maybe this method can be in the common module and be reused by both , the RMPage and the SchedulerPage ?